renovate.json

```diff
@@ -2,5 +2,12 @@
     "$schema": "https://docs.renovatebot.com/renovate-schema.json",
     "extends": [
         "config:recommended"
+    ],
+    "packageRules": [
+        {
```

Comment:
while we're here, can we add a rule to combine the PRs for the boto3* stuff?

Reply:
It probably doesn't matter since we're automerging but we can also just do this for all non-major deps. See [ami-deprecation-tool](https://github.com/canonical/ami-deprecation-tool/blob/main/renovate.json) for example

Reply:
better?